### PR TITLE
fixes typeable words and delay parameter for xdotool

### PR DESCRIPTION
**Pull request by c-nichols**
I ran into a couple issues while getting Caster to run on aenea:

- typeables was out of date with dragonfly. I added dragonfly's typeables to the set directly.
- kwargs needed to get passed through to dragonfly Key
- the delay parameter wasn't being passed correctly to xdotool

 let me know what you think of the fixes below, and thank you for putting together a fantastic project :-) I might be out of a job right now if it wasn't for the time and effort that you and others have put into making this possible.

### CODE REVIEW

File: client/aenea/config.py [hunk 1]
```diff
@@ -42,11 +42,13 @@
...
         'key_translations' : {'lessthan' : 'less', 'greaterthan' : 'greater'}, # Dragonfly expects us to use lessthan and greaterthan rather than less and greater.
         'modifiers': {'a': 'alt', 'A': 'Alt_R', 'c': 'control', 'w': 'super', 'h': 'hyper', 'm': 'meta', 'C': 'Control_R', 's': 'shift', 'S': 'Shift_R', 'M': 'Meta_R', 'W': 'Super_R', 'H': 'Hyper_R'},
         }
 
+_configuration['keys'] = list(set(_configuration['keys']).union(set(dragonfly.typeables.keys())))
```
Comment (calmofthestorm) [inline client/aenea/config.py]: Much better.

File: .gitignore [hunk 2]
```diff
@@ -3,3 +3,4 @@
 client/util/personal.py
 client/util/config.py
 server/linux_x11/config.py
+.idea
```
Comment (c-nichols) [inline .gitignore]: It was intentional -- I'm not sure how you (or most people) prefer to deal with IDE-specific files, I've always found it easiest to just ignore htem all. Will pull that out if you'd prefer --

**Comment by c-nichols** [comment]
Updated -- removed .idea from gitignore

**Comment by calmofthestorm** [comment]
Thank you!

**Pull request merged**